ui/packages/catalog-ui-search/src/main/webapp/component/list-create/list-create.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) Connexta, LLC
+ *
+ * <p>Unlimited Government Rights (FAR Subpart 27.4) Government right to use, disclose, reproduce,
+ * prepare derivative works, distribute copies to the public, and perform and display publicly, in
+ * any manner and for any purpose, and to have or permit others to do so.
+ */
+import React from 'react'
+import ListCreate from '../../react-component/presentation/list-create/list-create'
+import withListenTo from 'catalog-ui-search/src/main/webapp/react-component/container/backbone-container'
+
+var Marionette = require('marionette')
+
+const ListCreateWithListenTo = withListenTo(ListCreate)
```

Comment:
✏️ I think convention has been to export components with `withListenTo` if they need it.